docs/locales/ko_KR/LC_MESSAGES/common-api.po

```diff
@@ -33,14 +33,16 @@ msgid ""
 " <https://cloud.backend.ai>`_ or from the administrator of your "
 "Backend.AI cluster."
 msgstr ""
-"API서버에 요청하려면, 클라이언트에서는 API 엑세스 키와 시크릿키가 쌍으로 필요합니다`저희 클라우드 서비스 "
-"<https://cloud.backend.ai>`나 Backend.Ai 클러스터의관리자에서 해당 키를 확인할 수 있습니다."
+"API 서버에 요청하려면, 클라이언트에서는 API 엑세스 키와 비밀 키가 쌍으로 필요합니다. `저희 클라우드 서비스 "
+"<https://cloud.backend.ai>`_ 나 Backend.Ai 클러스터의 관리자에서 해당 키를 확인할 수 있습니다."
```

Comment:
Please change `Backend.Ai` to `Backend.AI` "AI" should be written in captials.
And also, we can remove '저희' from '저희 클라우드 서비스', since it's a redundant expression.